ports/raspberrypi/boards/pimoroni_inky_frame_5_7/mpconfigboard.h

```diff
@@ -4,7 +4,7 @@
 #define CIRCUITPY_DIGITALIO_HAVE_INVALID_PULL (1)
 #define CIRCUITPY_DIGITALIO_HAVE_INVALID_DRIVE_MODE (1)
 
-#define MICROPY_HW_LED_STATUS   (&pin_GPIO6)
+#define MICROPY_HW_LED_STATUS   (&pin_CYW0)
```

Comment:
I'd rather not change this because the built in LED isn't visible from the front.